packages/expo-cli/src/commands/codesigning/configureCodeSigningAsync.ts

```diff
@@ -0,0 +1,58 @@
+import {
+  convertCertificatePEMToCertificate,
+  convertKeyPairPEMToKeyPair,
+  validateSelfSignedCertificate,
+} from '@expo/code-signing-certificates';
+import { ExpoConfig, getConfig } from '@expo/config';
+import assert from 'assert';
+import { promises as fs } from 'fs';
+import path from 'path';
+
+import Log from '../../log';
+import { attemptModification } from '../utils/modifyConfigAsync';
+
+type Options = { input?: string };
+
+export async function actionAsync(projectRoot: string, { input }: Options) {
+  assert(typeof input === 'string', '--input must be a string');
+
+  const inputDir = path.resolve(projectRoot, input);
+
+  const [certificatePEM, privateKeyPEM, publicKeyPEM] = (
+    await Promise.all(
+      ['certificate.pem', 'private-key.pem', 'public-key.pem'].map(fname =>
+        fs.readFile(`${inputDir}/${fname}`)
```

Comment:
```suggestion
        fs.readFile(path.join(inputDir, fname), 'utf8')
```

Also the synchronous file methods often outperform the async ones except when working with large files. Don't think it matters either way here.